### PR TITLE
Import app

### PR DESCRIPTION
**Pull request by schleichardt**
still WIP but I create a pull request to be able to present a diff

### CODE REVIEW

**Comment by schleichardt** [comment]
@lauraluiz There is still a lot of work to do but I would like to get this branch merged and fix the stuff in smaller pull requests instead of letting this grow for ages.

Maybe we can sit together and collect items as well as prioritize.

File: importApp/src/main/java/com/commercetools/dataimport/products/ProductDraftReader.java [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+        if (productsCsvEntry.getImages().contains(";") || productsCsvEntry.getImages().contains(",")) {
+            throw new RuntimeException("not prepared for multi images" + productsCsvEntry.getImages());
+        }
+        if (!isEmpty(productsCsvEntry.getImages())) {
+            final String url = productsCsvEntry.getImages();
+            builder.images(Image.of(url, ImageDimensions.of(0, 0)));
+        }
+    }
+
+    private List<AttributeDraft> parseAttributes(final FieldSet currentLine, final List<ProductType> productTypes, final String productTypeKey) {
```
Comment (lauraluiz) [inline importApp/src/main/java/com/commercetools/dataimport/products/ProductDraftReader.java]: That's some massive method :D
Could you split it? Maybe it's already planned for the next improvements, so no need to do it in this PR.

**Comment by lauraluiz** [comment]
:+1:

File: importApp/src/main/java/com/commercetools/dataimport/products/ProductDraftReader.java (hunk 1 above)
Comment (schleichardt) [inline importApp/src/main/java/com/commercetools/dataimport/products/ProductDraftReader.java]: yep, I will do it later

**Pull request merged**